frontend/src/components/recipePage.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import './recipePage.css'
+
+export default function Recipe({name, image, ingredients, instructions}) {
+
+    // state variable that stores ingredient within RecipePage component    
+    const [newIngredient, setNewIngredient] = useState('');
+    const [newInstruction, setNewInstruction] = useState('');
+
+    let addIngredient = () => {
+        let Obj = {newIngredient: newIngredient}
```

Comment:
FYI there's a very useful shorthand in JS where writing`{ newIngredient }` is equivelent to writing `{ newIngredient : newIngredient }`